errors/tun.md

```diff
@@ -49,8 +49,28 @@ Thanks to [@Vendetta1985](https://github.com/Vendetta1985), [source comment](htt
 
 ## `TUN device is not available: open /dev/net/tun: permission denied`
 
-This can happen with `podman`.
-The only way known is to run the container with `--privileged`.
+This can happen with `podman`, usually due to SELinux. Create a SELinux policy to allow the rootless container to use the `/dev/net/tun` device.    
+- Copy the content below to a new file `gluetun_policy.te`
+```bash
+module gluetun_policy 1.0;
+
+require {
+        type tun_tap_device_t;
+        type container_file_t;
+        type container_t;
+        class chr_file { getattr ioctl open read write };
+        class sock_file watch;
+}
```

Comment:
It might be nice to have the policy as a standalone file in the repo with a perma link to it in the readme, so people can curl/get the file during setup instead of manually copy paste it 🙃